.floo

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I suggest you add the `.floo` and other files to your `.gitignore` file, to avoid committing it.

Reply:
Thank you Chris, I do need to know more about .gitignore .